velox/serializers/tests/PrestoSerializerTest.cpp

```diff
@@ -147,6 +149,18 @@ TEST_F(PrestoSerializerTest, basic) {
   testRoundTrip(rowVector);
 }
 
+TEST_F(PrestoSerializerTest, compress) {
```

Comment:
Can you parameterize this test file to cover each test case with two different modes: compressed and uncompressed? Thanks!